### PR TITLE
Add realistic names to dynamic loadout classes

### LINKED ISSUE #5184: Realistic Names for 1.70 (Aerial Vehicles)
**Arma 3 Version:** `1.70` (stable / rc / dev)
**CBA Version:** `3.3.1` (stable / dev + commit hash)
**ACE3 Version:** `3.9.2` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
- aerial vehicles, featuring the new dynamic loadout system, have changed their classnames. therefore they have no longer realistic names:

B_Plane_CAS_01_F ==> B_Plane_CAS_01_dynamicLoadout_F
B_Heli_Light_01_F ==> B_Heli_Light_01_dynamicLoadout_F
B_Heli_Attack_01_F ==> B_Heli_Attack_01_dynamicLoadout_F
B_UAV_02_F ==> B_UAV_02_dynamicLoadout_F
B_T_UAV_03_F ==> B_T_UAV_03_dynamicLoadout_F

O_Plane_CAS_02_F ==> O_Plane_CAS_02_dynamicLoadout_F
O_Heli_Light_02_F ==> O_Heli_Light_02_dynamicLoadout_F
O_Heli_Attack_02_F ==> O_Heli_Attack_02_dynamicLoadout_F
O_UAV_02_F ==> O_UAV_02_dynamicLoadout_F
O_T_VTOL_02_vehicle_F ==> O_T_VTOL_02_vehicle_dynamicLoadout_F
O_T_VTOL_02_infantry_F ==> O_T_VTOL_02_infantry_dynamicLoadout_F

I_Plane_Fighter_03_F ==> I_Plane_Fighter_03_dynamicLoadout_F
I_Heli_Light_03_F ==> I_Heli_Light_03_dynamicLoadout_F
I_UAV_02_F ==> I_UAV_02_dynamicLoadout_F

I_C_Heli_Light_01_Civil_F didn't change, but never had a realistic name. should be "MD-500".

**Steps to reproduce:**
- See list of assets

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) / Virtual Arsenal

**Placed Modules:**
-None

**RPT log file:**
- not needed

### PR DESCRIPTION
**Pull request by SilentSpike**
**When merged this pull request will:**
- Transfer existing realistic names to the new dynamic loadout classes (Fix #5184)

### CODE REVIEW

File: addons/realisticnames/CfgVehicles.hpp [hunk 1]
```diff
@@ -363,6 +392,11 @@ class CfgVehicles {
         displayName = CSTRING(Plane_Fighter_03_AA_Name);
     };
 
+    class Plane_Fighter_03_dynamicLoadout_base_F;
+    class I_Plane_Fighter_03_dynamicLoadout_F: Plane_Fighter_03_dynamicLoadout_base_F {
+        displayName = CSTRING(Plane_Fighter_03_Name);
+    };
+
     // uavs
     class UAV_02_base_F;
```
Comment (SilentSpike) [inline addons/realisticnames/CfgVehicles.hpp]: I was wondering while looking through the configs here, why are we not just adding the names to the base classes where applicable? This one for example, they all inherit from it anyway.

Comment (Dystopian) [inline addons/realisticnames/CfgVehicles.hpp]: because `displayName` is defined here.

Comment (SilentSpike) [inline addons/realisticnames/CfgVehicles.hpp]: It's not though, it's inherited from the base class :stuck_out_tongue:

Comment (Dystopian) [inline addons/realisticnames/CfgVehicles.hpp]: `air_f_gamma\Plane_Fighter_03\config.cpp`
```
class I_Plane_Fighter_03_dynamicLoadout_F: Plane_Fighter_03_dynamicLoadout_base_F {
	displayName = "$STR_A3_CfgVehicles_O_Plane_Fighter_03_CAS0";
```
did I miss anything?

Comment (SilentSpike) [inline addons/realisticnames/CfgVehicles.hpp]: The line I commented on :laughing:

Comment (SilentSpike) [inline addons/realisticnames/CfgVehicles.hpp]: For reference:
```
		class UAV_02_base_F: UAV //inherits 15 parameters from bin\config.bin/CfgVehicles/UAV, sources - ["A3_Drones_F_Air_F_Gamma_UAV_02"]
		{
			features = "Randomization: No						<br />Camo selections: 1 - the whole body						<br />Script door sources: None						<br />Script animations: None						<br />Executed scripts: None						<br />Firing from vehicles: No						<br />Slingload: Slingloadable						<br />Cargo proxy indexes: None";
			author = "Bohemia Interactive";
			_generalMacro = "UAV_02_base_F";
			displayName = "MQ-4A Greyhawk";
                        ...
		};
```

Comment (PabstMirror) [inline addons/realisticnames/CfgVehicles.hpp]: Vanilla Config:
`class UAV_02_base_F: UAV {displayName = "MQ-4A Greyhawk";}`
`class B_UAV_02_F: UAV_02_base_F` uses inheritance
`class O_UAV_02_F: UAV_02_base_F {displayName = "K40 Ababil-3";};`
 `class I_UAV_02_F: UAV_02_base_F {displayName = "K40 Ababil-3";}`
So adding it to all is necessary

**Pull request merged**